jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainGridLodControl.java

```diff
@@ -50,7 +50,7 @@ public TerrainGridLodControl(Terrain terrain, Camera camera) {
     }
     
     @Override
-    protected void updateLOD(List<Vector3f> locations, LodCalculator lodCalculator) {
+    protected void updateLOD(SafeArrayList<Vector3f> locations, LodCalculator lodCalculator) {
```

Comment:
where do you need it?
this is designed so that if you do a for(Object o:list) loop the iteration won't create an iterator but iterate over the array.